test/request_test.rb

```diff
@@ -145,17 +147,48 @@ class RequestTest < Minitest::Test
       end
     end
 
+    it "create the saml:AuthnContextClassRef element correctly" do
+      settings.authn_context = 'secure/name/password/uri'
+      auth_doc = OneLogin::RubySaml::Authrequest.new.create_authentication_xml_doc(settings)
+      assert auth_doc.to_s =~ /<saml:AuthnContextClassRef>secure\/name\/password\/uri<\/saml:AuthnContextClassRef>/
```

Comment:
It would be more clear _(and consistent)_ to use `assert_match` for all of these vs. `assert x =~ y`